release-notes-4.1.0.md

```diff
@@ -1,8 +1,13 @@
-!not-ready-for-release!
 
 #### Version Number
 ${version-number}
 
 #### New Features
+ - **SCMOD-10189**:  Updated Tomcat base image.  
+  This has changes to meet the logging mandate by adding caf logging Tomcat juli project to Tomcat Java 11 image. 
```

Comment:
Was the tomcat-juli project not in here a couple of releases ago?

Reply:
I looked at this commit https://github.com/CAFapi/opensuse-tomcat-cifs-image/commit/9e50cda84ffec330ddcdd6e7bd001c8658e0d5b8.

Reply:
But that version update had nothing to do with adding the juli project, it was fixing the logging issue. Look at the tomcat-jre11-image release notes and just copy that. Also the version of this project needs changed to a patch.